go/pkg/filemetadata/filemetadata.go

```diff
@@ -30,6 +31,26 @@ type FileError struct {
 	Err        error
 }
 
+type xattributeAccessorInterface interface {
+	getXAttr(path string, name string) ([]byte, error)
+}
+
+type xattributeAccessor struct{}
+
+func (x xattributeAccessor) getXAttr(path string, name string) ([]byte, error) {
+	return xattr.Get(path, name)
+}
+
+var (
+	XattrName   string
```

Comment:
wdyt of `XattrDigestName` or `XattrSHA256Name`? Since we're gating digest read on this one

Reply:
Also, maybe this should become a flag?

Reply:
I thought we were going to make this a constant?

Reply:
I think "DigestName", on the chance sometime in the far future someone decides SHA256 isn't good enough anymore.

Reply:
We discussed making it constant because of testing issues, but the issues exist anyway because I'm mocking the xattr package.  Since it didn't save any headache I put it back as a global so it's at least configurable, disabled by leaving it empty.